src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -201,6 +201,29 @@ struct GhValenciaDivCleanDefaults {
                                      grmhd::ValenciaDivClean::Tags::TildeS<>,
                                      grmhd::ValenciaDivClean::Tags::TildeB<>>>>;
 
+  enum class Phase {
```

Comment:
There's now a unified Phase class. Probably a bit of work to do when rebasing. Larry did this across a couple PRs

Reply:
I haven't changed this yet maybe we can go over this at some point,  will it mainly entail removing code?

Reply:
just get rid of the `enum class Phase` and the `phase_name` function.